src/UKNewsletterEpic/index.stories.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { ReactElement } from 'react';
+import { BrazeMessageComponent } from '../BrazeMessageComponent';
+import { StorybookWrapper } from '../utils/StorybookWrapper';
+import { withKnobs, text } from '@storybook/addon-knobs';
+
+export default {
+    component: 'UKNewsletterEpic',
+    title: 'WorkInProgress/EndOfArticle/UKNewsletterEpic',
+    decorators: [withKnobs({ escapeHTML: false })],
```

Comment:
As above, not sure we need the `escapeHTML` here.

Reply:
Without it we can't support the apostrophe in the title. It get's replaced with &apos;

Is there a better way to "unescape" this?

Reply:
Discussed offline - it sounds like we do need this. My bad, sorry!